pysatMissions/instruments/pysat_ephem.py

```diff
@@ -37,25 +38,29 @@ def init(self):
     Adds custom calculations to orbit simulation.
     This routine is run once, and only once, upon instantiation.
 
-    Adds quasi-dipole coordiantes, velocity calculation in ECEF coords,
-    adds the attitude vectors of spacecraft assuming x is ram pointing and
-    z is generally nadir, adds ionospheric parameters from the Interational
-    Reference Ionosphere (IRI), as well as simulated winds from the
-    Horiontal Wind Model (HWM).
+    Adds custom routines for quasi-dipole coordinates, velocity calculation in
+    ECEF coords, and attitude vectors of spacecraft (assuming x is ram pointing
+    and z is generally nadir).
 
     """
 
     self.custom_attach(mm_magcoord.add_quasi_dipole_coordinates)
     self.custom_attach(mm_magcoord.add_aacgm_coordinates)
     self.custom_attach(mm_sc.calculate_ecef_velocity)
     self.custom_attach(mm_sc.add_ram_pointing_sc_attitude_vectors)
```

Comment:
Should this be in preprocess?

Reply:
the functions need to be attached before the data is loaded

Reply:
Right, which is why I figured preprocess is where this belongs.  @rstoneback am I confused about this?

Reply:
fixed as discussed